### DISCUSSION TITLE
Get cell type percentage in each spot

### DISCUSSION THREAD

**Discussion by gambalab** [BayraktarLab/cell2location#93] [Q&A] [answered]
Hi, maybe is a stupid question, but is there a way to transform cell type abundance of each spot in percentage?

**Comment by vitkl** [comment, marked as answer]
Hi @gambalab 

You can compute that value yourself by taking cell abundance of all cell types (as in the tutorial plotting section), computing total cell abundance per location, and dividing values of individual cell types.
